forms-flow-api/src/formsflow_api/schemas/form_history_logs.py

```diff
@@ -16,3 +16,12 @@ class Meta:  # pylint: disable=too-few-public-methods
     created_by = fields.Str(data_key="createdBy")
     created = fields.Str(data_key="created")
     change_log = fields.Dict(data_key="changeLog")
+    major_version = fields.Str(data_key="majorVersion")
+    minor_version = fields.Str(data_key="minorVersion")
+    version = fields.Method("get_combined_version", dump_only=True)
+
+    def get_combined_version(self, obj):
+        """Combine major and minor versions."""
+        major_version = obj.major_version or "1"
```

Comment:
Can we use int instead of string "0" and "1" here ?

Reply:
done